shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -603,6 +654,48 @@ - (FlutterPlatformViewController*)platformViewController {
 
 #pragma mark - Private methods
 
+- (FlutterEngineViewRecord*)viewRecordForId:(uint64_t)viewId {
+  if (viewId == kFlutterDefaultViewId) {
+    return _viewRecord;
+  }
+  return nil;
+}
+
+- (void)updateSemantics:(const FlutterSemanticsUpdate*)update forViewId:(uint64_t)viewId {
+  FlutterEngineViewRecord* record = [self viewRecordForId:viewId];
+  if (record == nil) {
+    return;
+  }
+  std::shared_ptr<flutter::AccessibilityBridgeMac> bridge = record.bridge;
```

Comment:
The previous code asserted that the bridge had been created: https://github.com/flutter/engine/pull/38855/files#diff-9f979d53c3e186c5c7ab9e9f71924b50535d029f0f994ceafc21fb1f88011c01L921

Should we re-add that?

Reply:
Added it back.